project03.py

```diff
@@ -232,6 +232,28 @@ def verifyBirthBeforeDeath(person):
             return False
     return True
 
+<<<<<<< HEAD
+#aunts and uncles should not marry their neices and nephews
```

Comment:
ruh roh shaggy again but honestly we don't use this file anymore so delete it actually

Reply:
Like you can delete the whole file